plugin.go

```diff
@@ -169,7 +175,14 @@ func CopyTfEnv() {
 	}
 }
 
-func assumeRole(roleArn string) {
```

Comment:
I see we added a return of `bool` here.  But its never used and always will return `true`.  So I'm not sure we need to add a return type here.

Reply:
Actually for the line 178 below this line 😛

Reply:
The bool is just for the line 182, to avoid the assumeRole if one of the credentials have been set. Because of that, I needed to change the function signature and also returning true at the end of normal invocation. Do you have a suggestion?